debian/control

```diff
@@ -0,0 +1,42 @@
+Source: nfd
+Section: net
+Priority: optional
+Maintainer: Niv Sardi <xaiki@endlessm.com>
+Build-Depends: debhelper (>= 9),
+               dh-waf,
+               dh-python,
+               doxygen,
+               libboost-chrono-dev,
+               libboost-dev,
+               libboost-iostreams-dev,
+               libboost-log-dev,
+               libboost-program-options-dev,
+               libboost-regex-dev,
+               libboost-thread-dev,
+               libboost-random-dev,
+               libcrypto++-dev,
+               libndn-cxx-dev,
+               libsqlite3-dev,
+               libssl-dev,
+               libwebsocketpp-dev,
+               pkg-config,
+               python-sphinx,
+               python-sphinxcontrib.doxylink,
+               python | python-all | python-dev | python-all-dev,
+               python:any | python-all:any | python-dev:any | python-all-dev:any
+Standards-Version: 3.9.8
+Homepage: https://named-data.net
+Vcs-Git: https://github.com/named-data/NFD
+Vcs-Browser: https://github.com/named-data/NFD
+
+Package: nfd
+Architecture: any
+Depends: lsb-base (>= 3.0-6),
+         ndnsec-tools (= ${source:Version}),
```

Comment:
This is definitely broken. You can't expect that nfd's source version is going to match libndn-cxx's. Certainly in our case where the source version is calculated from git commits.